File: codigos.js.py/js_html/10c.js
/*Classe TV: Faça um programa que simule um televisor criando-o como um objeto.
 O usuário deve ser capaz de informar o número do canal e aumentar ou diminuir o volume. 
Certifique-se de que o número do canal e o nível do volume permanecem dentro de faixas válidas.*/ 

var tv ={
    canal: 10,
    volume: 16,

   mudarCanal: function(nCanal){
    if(this.canal >= 0 && this.canal <= 100)
    this.canal = nCanal;

    },
    mostrarCanal: function(){
        return this.canal;
   
       },
     mudarVolume: function(nVolume){
        if(this.volume >= 0 && this.volume <= 100)
        this.volume = nVolume;
    
        },
       
    
    mostrarVolume: function(){
        return this.volume;

    },

}

document.write("Canal = ", (tv.mostrarCanal()));
tv.mudarCanal(14);
document.write(" => Mudar para Canal = ", (tv.mostrarCanal()));
document.write(" => Volume = ", (tv.mostrarVolume()));
tv.mudarVolume(18);
document.write(" => Aumentar volume = ", (tv.mostrarVolume()));
tv.mudarVolume(12);
document.write(" => Diminuir volume = ", (tv.mostrarVolume()));

File: codigos.js.py/scripts_py/10c.py
'''Classe TV: Faça um programa que simule um televisor criando-o como um objeto.
 O usuário deve ser capaz de informar o número do canal e aumentar ou diminuir o volume. 
Certifique-se de que o número do canal e o nível do volume permanecem dentro de faixas válidas.'''

class tv:
    def __init__(self, canal, volume):
        self.canal = canal
        self.volume = volume

    def mudar_canal(self, n_canal):
        if self.canal >= 0 and self.canal <= 100:
            self.canal = n_canal


    def mostrar_canal(self):
        return self.canal

    def mudar_volume(self, n_volume):
        if self.volume >= 0 and self.volume <= 100:
            self.volume = n_volume

    def mostrar_volume(self):
        return self.volume

t = tv(10, 15)
print("\nCanal = ", t.mostrar_canal())
t.mudar_canal(21)
print("Novo canal = ", t.mostrar_canal())
print("Volume = ", t.mostrar_volume())
t.mudar_volume(18)
print("Aumentar volume = ", t.mostrar_volume())
t.mudar_volume(10)
print("Diminuir volume = ", t.mostrar_volume())
print("")


        
	 
